package.json

```diff
@@ -278,7 +278,8 @@
       "scheduler": "0.25.0-rc-372ec00c-20241209"
     },
     "patchedDependencies": {
-      "webpack-sources@3.2.3": "patches/webpack-sources@3.2.3.patch"
+      "webpack-sources@3.2.3": "patches/webpack-sources@3.2.3.patch",
+      "@storybook/react@8.4.7": "patches/@storybook__react@8.4.7.patch"
```

Comment:
Patched `@storybook/react` for type error:

- https://github.com/vercel/next.js/actions/runs/12394130672/job/34596896487#step:8:565

Opened issue at Storybook: https://github.com/storybookjs/storybook/issues/30104

cc @eps1lon